app/src/components/main/analytics/events/ga4Events/profilePageEvent.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const PROFILE_PAGE = 'profile';
+
+const BASIC_EVENT_PARAMETERS_PROFILE = {
+  action: 'click',
+  category: PROFILE_PAGE,
+  element_name: 'language_drop_down',
```

Comment:
element_name can change very frequently. That is why I suppose to put that field in every event instead of putting it in BASIC_EVENT_PARAMETERS object. Or we can create a function, which returns BASIC_EVENT_PARAMETERS object.

Reply:
fixed ✅